tests/integration_tests/modules/test_combined.py

```diff
@@ -514,7 +514,7 @@ def test_instance_json_gce(self, class_client: IntegrationInstance):
         # attributes
         assert isinstance(client.instance, GceInstance)
         assert v1_data["availability_zone"] == client.instance.zone
-        assert v1_data["instance_id"] == client.instance.instance_id
+        assert v1_data["instance_id"] == str(client.instance.id)
```

Comment:
Since the pycloudlib branch landed, we don't need to enclosing str()
Let's just bump the pycloudlib version to `pycloudlib>=1!9.1.0,<1!11` in integration-requirements.txt cloud-init testing shouldn't be affected by  the recent drop of py36 support that forced the pycloudlib VERSION bump to 1!-10